lucupy/minimodel/timeallocation.py

```diff
@@ -44,6 +44,13 @@ class TimeAllocation:
     and the time that has been used, divided between program time and partner
     calibration time. The time used is calculated as a ratio of the awarded time
     for this category to the total time awarded to the program.
+
+    Attribute:
+        category (TimeAccountingCode):
+        program_awarded (timedelta):
+        partner_awarded (timedelta):
+        program_used (timedelta):
+        partner_used (timedelta):
```

Comment:
Do we want descriptions here?
* category: The account from which the time has been awarded (country or otherwise).
* program_awarded: The amount of program time that was awarded.
* partner_awarded: The amount of partner calibration time that was awarded.
* program_used: The amount of program_awarded time used so far.
* partner_used: The amount of partner_awarded time used so far.